bazel_tools/haskell_public_ghci_repl_wrapper.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/haskell/repl.bzl b/haskell/repl.bzl
+index 4746d3c..2c836f7 100644
+--- a/haskell/repl.bzl
++++ b/haskell/repl.bzl
+@@ -296,7 +296,7 @@ def _create_repl(hs, ctx, repl_info, output):
```

Comment:
This looks reasonable for upstreaming - I assume that's the plan?

Reply:
Yep, just wanted to get something working for now.